pkg/virt-handler/vm.go

```diff
@@ -538,7 +538,7 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 		if len(vmi.Status.VolumeStatus) > 0 {
 			diskDeviceMap := make(map[string]string)
 			for _, disk := range domain.Spec.Devices.Disks {
-				diskDeviceMap[strings.TrimPrefix(disk.Alias.Name, api.UserAliasPrefix)] = disk.Target.Device
+				diskDeviceMap[strings.TrimPrefix(disk.Alias.GetName(), api.UserAliasPrefix)] = disk.Target.Device
```

Comment:
This trim should not be necessary anymore.

Reply:
This invocation of TrimPrefix is removed.